enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/cluster/ClusterState.java

```diff
@@ -299,9 +299,11 @@
                                     if ( discoveredInstance.getJoiningId().equals( configurationRequested.getJoiningId() ) )
                                     {
                                         // we are done
-                                        StringBuffer errorMessage = new StringBuffer( "Failed to join cluster because I saw two instances with the same ServerId" );
-                                        errorMessage.append( "One is " ).append( discoveredInstance.getJoiningId() );
-                                        errorMessage.append( " The other is " ).append( configurationRequested );
+                                        StringBuffer errorMessage = new StringBuffer(
```

Comment:
StringBuffer is probably an overkill. Simple string concatenation should be equally performant and more readable.